### PR TITLE
Renaming errors, reorganizing tests

### LINKED ISSUE #17: Add missing test paths to unit tests
Make sure each line of code is covered by at least one unit test. Including emitted events.

### PR DESCRIPTION
**Pull request by microbecode**
- Rename error messages
- Reorganize unit tests to logical places. Add a few missing tests
- Restart the devnet before each test section

Closes #17

### CODE REVIEW

**Comment by JaniAnttonen** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>f03ba82</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://starknet-multisig-e4bcv29i2-equilibriumco.vercel.app'>https://starknet-multisig-e4bcv29i2-equilibriumco.vercel.app</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/equilibriumco/starknet-multisig/6ASUbszgJtXr3AKsHKH7Vz4a3Tcy'>https://vercel.com/equilibriumco/starknet-multisig/6ASUbszgJtXr3AKsHKH7Vz4a3Tcy</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/eqlabs/starknet-multisig/actions/runs/2704434021)

**Comment by microbecode** [comment]
> Some illogical naming here and there (using the nonSigner as a signer)

Hmm. Well, at the beginning of the test "nonSigner" is not a signer. He is then assigned as a signer and then can be used as signer throughout the test.
Should I deploy a new Account who can be added as a signer? What should I call it?

File: contracts/test/Multisig.test.ts [hunk 1]
```diff
@@ -68,272 +70,166 @@ describe("Multisig with single signer", function () {
...
-      expect(res.tx.threshold).to.equal(0n);
-      expect(res.tx_calldata_len).to.equal(1n);
-      expect(res.tx_calldata[0]).to.equal(5n);
-    });
+      const signers = await newMultisig.call("get_signers");
+      const signers_length = await newMultisig.call("get_signers_len");
+      const isSignerValid1 = await newMultisig.call("is_signer", {
+        address: number.toBN(account.starknetContract.address),
+      });
+      const isSignerValid2 = await newMultisig.call("is_signer", {
```
[the remark this answers is not in the document]
Comment (microbecode) [reply contracts/test/Multisig.test.ts:90]: ping (about my comment)

[the remark this answers is not in the document]
Comment (JaniAnttonen) [reply contracts/test/Multisig.test.ts:90]: A nice bypass would be to create an alias for this test only, and maybe comment what the alias is there for.

**Comment by JaniAnttonen** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>93b9572</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://starknet-multisig-dyhp2zwo8-equilibriumco.vercel.app'>https://starknet-multisig-dyhp2zwo8-equilibriumco.vercel.app</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/equilibriumco/starknet-multisig/6jwcFuEdWmZ4CqJAr7PE4U82ddDB'>https://vercel.com/equilibriumco/starknet-multisig/6jwcFuEdWmZ4CqJAr7PE4U82ddDB</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/eqlabs/starknet-multisig/actions/runs/2737614245)

File: contracts/test/Multisig.test.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (microbecode) [reply contracts/test/Multisig.test.ts:91]: Hmm this is a valid idea. I now added an extra variable for all tests which make 'nonSigner' a signer, which should make it more clear.

**Review by JaniAnttonen — APPROVED** [review]

**Pull request merged**